docs/tutorial/fdfitting.rst

```diff
@@ -36,24 +33,24 @@ Entering model prints the model equation and its default parameters::
 
     Name      Value  Unit      Fitted      Lower bound    Upper bound
     ------  -------  --------  --------  -------------  -------------
-    DNA/Lp    40     [nm]      True                  0            inf
-    DNA/Lc    16     [micron]  True                  0            inf
-    DNA/St  1500     [pN]      True                  0            inf
-    kT         4.11  [pN*nm]   False                 0              8
+    DNA/Lp    40     [nm]      True              0.001            100
+    DNA/Lc    16     [micron]  True            0.00034            inf
+    DNA/St  1500     [pN]      True                  1            inf
+    kT         4.11  [pN*nm]   False              3.77              8
 
-Note how most of the parameters listed have the model name prefixed. For example, the persistence length in this model
-is now named `DNA/Lp`. Here `DNA` refers to the name we gave to the model.
+As we can see from the model equation, the model is given as distance as a function of force.
+Note also how the model name is prefixed to the model specific parameters, forming a key-value pair like so: `"name/parameter": value`. For example, the persistence length in this model
+is now named `DNA/Lp`.
 
-One parameter stands out, which is kT as it doesn't have `DNA` prefixed. The reason for this is that this parameter is
-not model specific (in this case the parameter represents the Boltzmann constant times the temperature).
+kT does not have `DNA` prefixed, because this parameter is not model specific (the parameter represents the Boltzmann constant times the temperature).
```

Comment:
```suggestion
kT does not have `DNA` prefixed since it represents the product of the Boltzmann constant times the temperature, and is therefore not model specific.
```